dolphin-dnn/src/test/java/edu/snu/dolphin/dnn/blas/function/AbsoluteTest.java

```diff
@@ -44,10 +44,19 @@ public void setup() throws InjectionException {
         .build();
     final MatrixFactory matrixFactory = Tang.Factory.getTang().newInjector(conf).getInstance(MatrixFactory.class);
 
-    this.input = matrixFactory.create(new float[][]{{1.0f, -2.0f, 3.0f}, {-4.0f, 5.0f, -6.0f}});
-    this.expectedOutput = matrixFactory.create(new float[][]{{1.0f, 2.0f, 3.0f}, {4.0f, 5.0f, 6.0f}});
-    this.expectedDerivative = matrixFactory.create(new float[][]{
-        {1.0f, -1.0f, 1.0f}, {-1.0f, 1.0f, -1.0f}});
+    final int numInput = 3;
+    final int numBatch = 2;
+    this.input = matrixFactory.create(new float[]{
+        1.0f, -2.0f, 3.0f,
+        -4.0f, 5.0f, -6.0f},
+        numInput, numBatch);
```

Comment:
Since a single input instance is stored in each column, it seems not straightforward in Java representation if we use `create(float[][])`.
For example, we should use `create(new float[][]{{1.0f, -4.0f}, {-2.0f, 5.0f}, {3.0f, -6.0f}}` for generating `input` that consists of two input instances `(1, -2, 3)` and `(-4, 5, -6)`.
However, multiple instances can be distinguishably represented with line breaks when we use `create(float[], int, int)`.